modules/apps/asset/asset-categories-selector-web/src/main/resources/META-INF/resources/js/SelectCategory.es.js

```diff
@@ -62,43 +63,47 @@ function SelectCategory({
 
 	const selectedNodesRef = useRef(null);
 
+	const openingWindow = Liferay.Util.getOpener();
```

Comment:
All changes in this file should be done alongside migration of [this `ItemSelectorDialog` in asset-taglib](https://github.com/liferay/liferay-portal/blob/master/modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_categories_selector/AssetVocabularyCategoriesSelector.es.js#L143). Markup of new dialog might be different.

It's probably best to send the changes in this file along with LPS-129589, and then followup with the rest of this PR.

Reply:
Ok that sounds good.  I will move these changes to `asset` and start working on that.